### PR TITLE
Chore: Refactor _get_admin_password to property _admin_password

### PR DESCRIPTION
**Pull request by ghislainbourgeois**

### CODE REVIEW

File: orchestrator-bundle/nms-magmalte-operator/src/charm.py [hunk 1]
```diff
@@ -427,14 +427,14 @@ def _on_get_master_admin_credentials(self, event: ActionEvent) -> None:
             if not self.model.get_relation("replicas"):
                 event.fail("Peer relation not created yet")
                 return
-            password = self._get_admin_password()
+            password = self._admin_password
             if not password:
```
Comment (gruyaume) [inline orchestrator-bundle/nms-magmalte-operator/src/charm.py]:
```suggestion
            if not self._admin_password:
```

File: orchestrator-bundle/nms-magmalte-operator/src/charm.py [hunk 2]
```diff
@@ -427,14 +427,14 @@ def _on_get_master_admin_credentials(self, event: ActionEvent) -> None:
             if not self.model.get_relation("replicas"):
                 event.fail("Peer relation not created yet")
                 return
-            password = self._get_admin_password()
+            password = self._admin_password
```
Comment (gruyaume) [inline orchestrator-bundle/nms-magmalte-operator/src/charm.py]:
```suggestion
```

File: orchestrator-bundle/nms-magmalte-operator/src/charm.py [hunk 3]
```diff
@@ -306,7 +306,7 @@ def _create_master_nms_admin_user(self) -> None:
         Returns:
             None
         """
-        password = self._get_admin_password()
+        password = self._admin_password
         if not password:
```
Comment (gruyaume) [inline orchestrator-bundle/nms-magmalte-operator/src/charm.py]:
```suggestion
        if not self._admin_password:
```

File: orchestrator-bundle/nms-magmalte-operator/src/charm.py [hunk 4]
```diff
@@ -306,7 +306,7 @@ def _create_master_nms_admin_user(self) -> None:
         Returns:
             None
         """
-        password = self._get_admin_password()
+        password = self._admin_password
```
Comment (gruyaume) [inline orchestrator-bundle/nms-magmalte-operator/src/charm.py]:
```suggestion
```

**Review by gruyaume — APPROVED** [review]
Minor details around not creating variables and pointing to property instead. You can ignore if you want.

File: orchestrator-bundle/nms-magmalte-operator/src/charm.py (hunk 1 above)
Comment (ghislainbourgeois) [reply orchestrator-bundle/nms-magmalte-operator/src/charm.py]: Done

File: orchestrator-bundle/nms-magmalte-operator/src/charm.py (hunk 2 above)
Comment (ghislainbourgeois) [reply orchestrator-bundle/nms-magmalte-operator/src/charm.py]: Done

File: orchestrator-bundle/nms-magmalte-operator/src/charm.py (hunk 4 above)
Comment (ghislainbourgeois) [reply orchestrator-bundle/nms-magmalte-operator/src/charm.py]: Done

File: orchestrator-bundle/nms-magmalte-operator/src/charm.py (hunk 3 above)
Comment (ghislainbourgeois) [reply orchestrator-bundle/nms-magmalte-operator/src/charm.py]: Done

**Review by gruyaume — APPROVED** [review]

**Pull request merged**